surveyor/collector_statz.go

```diff
@@ -567,6 +575,24 @@ func (sc *StatzCollector) pollAccountInfo() error {
 				consumerCount: float64(len(stream.Consumer)),
 				replicaCount:  float64(stream.Config.Replicas),
 			})
+
+			// computed tiered storage usage
+			// todo: figure out if we need to multiply size * replicas here
+			// size := float64(stream.State.Bytes * uint64(stream.Config.Replicas))
+			size := float64(stream.State.Bytes)
```

Comment:
need to test to figure out which size calculation to use

Reply:
Okay, I have almost everything working, we will need to do the multiplication based on replica. I'll update this pr